src/scripts/printToDOM.js

```diff
@@ -0,0 +1,3 @@
+const printToDOM = (object) => {
+    document.querySelector("#this_can_be_called_whatever_for_now").innerhtml += object
```

Comment:
shouldn't this function also take the location inside queryselector as an argument?

Reply:
w/e panyyaaaa